games/y.yaml

```
@@ -5,9 +5,9 @@
   development: active
   originals:
   - Quake 2
-  repo: https://github.com/yquake2
+  repo: https://github.com/yquake2/yquake2
   type: remake
-  updated: 2015-04-05
+  updated: 2019-09-07
```

```suggestion
  updated: 2015-04-05
```
Since nothing really changed, leave the updated date as-is
Don't worry about dangerJS warnings, they are warnings only and meant for game additions/updates